app/controllers/api/projects.php

```diff
@@ -1006,6 +1006,7 @@
 
         $task = new Document([
             '$id' => $dbForConsole->getId(),
+            '$projectId' => $project->getId(),
```

Comment:
Can remove the `$` prefix (used the `$collection` attribute from the Document model as inspiration):
https://github.com/appwrite/appwrite/blob/feat-db-refactor-routes/src/Appwrite/Utopia/Response/Model/Document.php#L38-L43

However, we do need to pass the `projectID` to the `tasks` worker because tasks are stored at the project level:
https://github.com/appwrite/appwrite/blob/feat-db-refactor-routes/app/workers/tasks.php#L70

Reply:
`$` removed in c04e2ba